CIP-0142/README.md

```diff
@@ -0,0 +1,114 @@
+---
+CIP: 0142
+Title: Web-Wallet Bridge - Network Determination
+Status: Proposed
+Category: Wallets
+Authors:
+  - Steven Johnson <steven.johnson@iohk.io>
+  - Nathan Bogale <nathan.bogale@iohk.io>
+Implementors:
+  - Lace <https://www.lace.io/>
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/209
+  - https://github.com/cardano-foundation/CIPs/pull/323
+  - https://github.com/cardano-foundation/CIPs/pull/960
```

Comment:
```suggestion
Discussions:
  - https://github.com/cardano-foundation/CIPs/pull/209
  - https://github.com/cardano-foundation/CIPs/pull/323
  - https://github.com/cardano-foundation/CIPs/pull/960
  - https://github.com/cardano-foundation/CIPs/pull/962
```

Reply:
will be resolved when https://github.com/cardano-foundation/CIPs/pull/972#issuecomment-2610505705 is addressed.  Resulting `Discussions:` should have both #960 and #972.